.babelrc

```diff
@@ -1,20 +1,45 @@
 {
-    "presets": [
-        ["es2015", {
-            "modules": false
-        }],
-        "flow"
-    ],
-    "plugins": [
-        "transform-runtime",
-        "transform-class-properties",
-        "version-inline"
-    ],
+    "presets": ["flow"],
     "env": {
+        // Duplications are inevitable due to strange .babelrc env merging:
+        // babel/babel#5276. This can be simplified in Babel 7 using .babelrc.js.
+        "production": {
+            "presets": [["es2015", {"modules": false}]],
+            "plugins": [
+                "transform-class-properties",
+                "transform-runtime",
+                "version-inline",
+            ],
+        },
+        "development": {
+            "presets": [["es2015", {"modules": false}]],
+            "plugins": [
+                "transform-class-properties",
+                "transform-runtime",
+                "version-inline",
+            ],
+        },
         "test": {
+            "presets": ["es2015"],
+            "plugins": [
+                "transform-class-properties",
+                "transform-runtime",
+                "version-inline",
+            ],
+        },
+        "esm": {
             "plugins": [
-                "transform-es2015-modules-commonjs"
+                "transform-class-properties",
+                "version-inline",
+                ["import-rename", {
+                    // source files and font metrics data are moved to lib/
+                    "^\\./src/": "./lib/",
```

Comment:
Sneaky.

Reply:
@kevinbarabash Yes, but it uses AST and I didn't want to use another module bundler.